libobs/obs-internal.h

```diff
@@ -953,6 +953,22 @@ static inline void obs_source_dosignal(struct obs_source *source,
 				      &data);
 }
 
+static inline void obs_transition_dosignal(struct obs_source *source,
+					   const char *signal_obs,
+					   const char *signal_source)
+{
+	struct calldata data;
+	uint8_t stack[128];
+
+	calldata_init_fixed(&data, stack, sizeof(stack));
+	calldata_set_ptr(&data, "source", source);
+	if (signal_source) {
+		signal_handler_signal(obs->signals, signal_obs, &data);
+		signal_handler_signal(source->context.signals, signal_source,
+				      &data);
+	}
+}
```

Comment:
Assuming you just wanted to ignore `source->context.private` from `obs_source_dosignal`, the signal names should be checked for each.
```suggestion
	if (signal_obs)
		signal_handler_signal(obs->signals, signal_obs, &data);
	if (signal_source)
		signal_handler_signal(source->context.signals, signal_source,
				      &data);
}
```

Reply:
Indeed that was an oversight, thank you.